CHANGELOG.md

```diff
@@ -44,6 +44,7 @@
 - Minor: Mod list, VIP list, and Users joined/parted messages are now searchable. (#3426)
 - Minor: Add search to emote popup. (#3404)
 - Minor: Messages can now be highlighted by subscriber or founder badges. (#3445)
+- Minor: Add workaround for multipart emoji (#3469)
```

Comment:
Make this a beefier changelog entry with a link to the public RFC/PDF thing hosted on a non-private domain somewhere